logic/src/commonJvmAndroid/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -153,7 +153,7 @@ class CallManagerImpl internal constructor(
                     messageSender,
                     scope,
                     callMapper
-                ).keepingStrongReference(),
+                ),
```

Comment:
Why are we not `keepingStrongReference` of `OnSendOTR` handler? 🤔

Reply:
Oh! that should not be removed. Good catch 👌